docs/api-docs/index.md

```diff
@@ -40,7 +40,7 @@ Explore the App Center API Service with [swagger](https://openapi.appcenter.ms).
 
 	![Setting api token value to authorize our API's](~/api-docs/images/authorization_withtoken.PNG)
 
-13. On the top right corner of the pop up, click the "X" to exit the pop up.
+13. If it shows 'Authorized' and a Logout-Button, authorization was successfull. On the top right corner of the pop up, click the "X" to exit the pop up.
```

Comment:
Thanks for catching this @K232 !
Spelling check - "successful" instead of "successfull". For consistency, could you use double inverted commas for "Authorized" and bold **Logout** (i.e. If it shows "Authorized" and a **Logout** button).